apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -29,7 +29,8 @@ const styles = {
     margin: '0 0 0 7px'
   },
   dropdown: {
-    margin: '0 6px'
+    margin: '0 6px',
+    width: 300
```

Comment:
The dropdown was not wide enough to see all the text to I made it wider while I was here.